### PR TITLE
 (enhancement) use showModal function to display start-visit-modal

### PR DESCRIPTION
**Pull request by donaldkibet**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide).
- [x] My work includes tests or is validated by existing tests.

## Summary
1. This is a follow-up PR for updating how modals are displayed using the `showModal` function. I have added a test for the changes made.
2. Renamed `form-overview.component` to `forms-summary-dashboard.component`

### CODE REVIEW

File: packages/esm-patient-forms-app/src/form-entry-interop.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { navigate, Visit } from '@openmrs/esm-framework';
+import { navigate, showModal, Visit } from '@openmrs/esm-framework';
```
Comment (brandones) [inline packages/esm-patient-forms-app/src/form-entry-interop.ts:1]:
```suggestion
import { navigate, Visit } from '@openmrs/esm-framework';
```

File: packages/esm-patient-forms-app/src/index.ts [hunk 2]
```diff
@@ -55,7 +55,7 @@ function setupOpenMRS() {
         name: 'forms-widget',
         slot: 'patient-chart-summary-dashboard-slot',
         order: 5,
-        load: getAsyncLifecycle(() => import('./forms/forms-overview.component'), options),
```
Comment (brandones) [inline packages/esm-patient-forms-app/src/index.ts:58]: Is `forms-overview.component` unused? Can we delete it?

**Review by brandones — APPROVED** [review]
A couple points of feedback, but generally looks good!

File: packages/esm-patient-forms-app/src/index.ts (hunk 2 above)
Comment (donaldkibet) [reply packages/esm-patient-forms-app/src/index.ts:58]: Its used renamed it to `form-summary-dashboard.component.tsx`

**Pull request merged**